10LessonTen/Checkpoint2/index.html

```diff
@@ -1,10 +1,116 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Syn Music</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
   <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/css-reset.css">
```

Comment:
It would be best to put your css-reset above your main style sheet. You could even put all the CSS reset code at the very top of your main style sheet.